pkg/bitcoin/block.go

```diff
@@ -69,3 +105,70 @@ func (bh *BlockHeader) Hash() Hash {
 	//       3. Construct the Hash instance appropriately.
 	return Hash{}
 }
+
+// Target calculates the difficulty target of a block header.
+func (bh *BlockHeader) Target() *big.Int {
+	// A serialized 80-byte block header stores the `bits` value as a 4-byte
+	// little-endian hexadecimal value in a slot including bytes 73, 74, 75, and
+	// 76. This function's input argument is expected to be a numerical
```

Comment:
Which function's ?

Reply:
I replaced this lengthy explanation of how target is calculated with a short function description.
We relay on the `btcsuite` library now.